src/bitcoin-cli.cpp

```diff
@@ -477,7 +477,7 @@ class NetinfoRequestHandler : public BaseRequestHandler
             const int8_t network_id{NetworkStringToId(network)};
             if (network_id == UNKNOWN_NETWORK) continue;
             const bool is_outbound{!peer["inbound"].get_bool()};
-            const bool is_block_relay{!peer["relaytxes"].get_bool()};
+            const bool is_tx_relay{peer["relaytxes"].isNull() ? true : peer["relaytxes"].get_bool()};
```

Comment:
Isn't this *wrong*?

Reply:
Can you elaborate?

Reply:
If relaytxes is missing, it's not necessarily true - could just as well be false.

Reply:
This discussion made me look into a small quirk I observed while reviewing this PR:

Sometimes I would see a `*` displayed for a peer that just connected, which would vanish with the next refresh of -netinfo.
This is because when we make an outbound connection, have sent a version message, but haven't received an answer, the node hasn't initialized TxRelay because it doesn't have that info yet, so we'll get false as an answer ([code](https://github.com/bitcoin/bitcoin/blob/b4f686952a60bbadc7ed2250651d0d6af0959f4d/src/net_processing.cpp#L1421)).
So if we'd want to be extremely correct, the wording could be changed to "the peer didn't request we relay transactions to it (relaytxes is false)".

Reply:
@mzumsande ACK

Reply:
> This discussion made me look into a small quirk I observed while reviewing this PR:
> 
> Sometimes I would see a `*` displayed for a peer that we just connected to, which would vanish with the next refresh of -netinfo. This is because when we make an outbound connection, have sent a version message, but haven't received an answer, the node hasn't initialized TxRelay because it doesn't have that info yet, so we'll get false as an answer ([code](https://github.com/bitcoin/bitcoin/blob/b4f686952a60bbadc7ed2250651d0d6af0959f4d/src/net_processing.cpp#L1421)). So if we'd want to be extremely correct, the wording could be changed to "the peer didn't request we relay transactions to it (relaytxes is false)".

@mzumsande added a commit authored by you in #25923